applications/matter_bridge/README.rst

```diff
@@ -270,6 +278,32 @@ You can enable the :ref:`matter_bridge_app_bridged_support` by using the followi
 * :kconfig:option:`CONFIG_BRIDGED_DEVICE_SIMULATED` - For the simulated bridged device.
 * :kconfig:option:`CONFIG_BRIDGED_DEVICE_BT` - For the Bluetooth LE bridged device.
 
+Additionally, you can decide how many bridged devices the bridge application will support.
+The decision will make an impact on the flash and RAM memories usage, what can be verified on the compilation level.
+The application uses dynamic memory allocation and stores bridged devices objects on heap, so it may be necessary to increase the heap size using ::kconfig_option::`CONFIG_CHIP_MALLOC_SYS_HEAP_SIZE`.
+Use the following configuration options to customize the number of supported bridged devices:
+
+* ::kconfig_option::`CONFIG_BRIDGE_MAX_BRIDGED_DEVICES_NUMBER` - For changing the maximum number of non-Matter bridged devices supported by the bridge application
+* ::kconfig_option::`CONFIG_BRIDGE_MAX_DYNAMIC_ENDPOINTS_NUMBER` - For changing the maximum number of Matter endpoints used for bridging devices by the bridge application.
+  This option does not have to be equal to ::kconfig_option::`CONFIG_BRIDGE_MAX_BRIDGED_DEVICES_NUMBER`, as it is possible to use non-Matter devices that are represented using more than one Matter endpoints.
+
+Configuring the number of Bluetooth LE bridged devices
+------------------------------------------------------
+
+For the Bluetooth LE bridged devices, you have to consider more factors before deciding about the number of supported bridged devices.
+Every Bluetooth LE bridged devices uses a separate Bluetooth LE connection, so you should increase the number of devices in a consistent way with the number of supported Bluetooth LE connections selected using ::kconfig_option::`CONFIG_BT_MAX_CONN`.
+Additionally, the Matter stack uses one Bluetooth LE connection for the commissioning, so at most it is possible to use ::kconfig_option::`CONFIG_BT_MAX_CONN` - 1 for the bridged devices purpose.
```

Comment:
```suggestion
Since the Matter stack uses one Bluetooth LE connection for commissioning, the maximum number of connections you can use for bridged devices is one less than is configured using the :kconfig:option:`CONFIG_BT_MAX_CONN` Kconfig option.
```